### ISSUE TITLE
texlive install fails

### ISSUE THREAD

**Issue by benneti** [termux/termux-app#305]
Running apt install texlive fails.
```

Unpacking texlive (20160523b-1) ...
Processing triggers for man (1.14.1-1) ...
Setting up texlive (20160523b-1) ...
========================================================
retrieving texlive...
you can start this manually by calling termux-install-tl
========================================================
Connecting to mirror.ctan.org (176.28.54.184:80)
Connecting to ftp.mpi-sb.mpg.de (139.19.205.14:21)
install-tl-unx.tar.g 100% |*******|  3258k  0:00:00 ETA
Automated TeX Live installation using profile: texlive_inst.profile
./install-tl: Initialization failed (in setup_unix_one):
./install-tl: could not find a usable wget.
./install-tl: Please install wget and try again.
./install-tl: Goodbye.
sed: /data/data/com.termux/files/usr/opt/texlive/2016/tlpkg/TeXLive/TLUtils.pm: No such file or directory
/data/data/com.termux/files/usr/bin/termux-install-tl[69]: texlinks: not found
```

The message suggegsts what is missingbut it is actually available
EDIT: If this should be an issue of another package I am sorry and also thank you for the great app it makes my Phone so much cooler 👍

**Comment by Grimler91** [comment]
`packages install wget` should solve the problem. Do you really have wget available?

**Comment by benneti** [comment]
OK Sorry, it was the version from busybox not from the repos. With the one from the repos it does indeed work.
Thank you

**Comment by Grimler91** [comment]
Great that it works! 

Thanks for you bug report. I thought I fixed this in [#873](https://github.com/termux/termux-packages/pull/873) (so that busybox's wget would suffice) but apparently not.
